source/develop/release-management/features/network/liveMigrationSupportForSRIOV.html.md

```diff
@@ -0,0 +1,83 @@
+---
+title: LiveMigrationSupportForSRIOV
+category: feature
+authors: mmucha
+wiki_category: Feature
+wiki_title: Features/LiveMigrationSupportForSRIOV
+wiki_revision_count: 12
+wiki_last_updated: 2015-10-14
+feature_name: Live Migration Support For SRIOV
+feature_modules: Networking
+feature_status: Design
+rfe: https://bugzilla.redhat.com/show_bug.cgi?id=868811
+
+---
+
+
+# Live Migration Support For SRIOV
+
+### Owner
+
+*   Name: [ Martin Mucha](User:mmucha)
+*   Email: mmucha@redhat.com
+
+## Summary
+Current support of SR-IOV in engine does not allow migration, which
```

Comment:
Can you expand shortcuts on first usage?
SR-IOV (single root I/O virtualization)
VF (virtual function)
....